includes/wikia/models/DesignSystemGlobalNavigationModelV2.class.php

```diff
@@ -60,11 +61,15 @@ public function getData() {
 		return $data;
 	}
 
-	private function getMainNavigation() {
-		return array_merge(
-			$this->getFandomLinks(),
-			[ $this->getWikisMenu() ]
-		);
+	private function getMainNavigation( $isWikiaOrgCommunity ) {
+		if ( $isWikiaOrgCommunity ) {
+			return [$this->getLink( self::COMMUNITY_CENTRAL_LABEL, $this->getHref('community-central'), self::COMMUNITY_CENTRAL_TRACKING_LABEL )];
```

Comment:
This line is super long, can we break it up?

Also, what do you think about removing the else from this method or converting it to a ternary?